src/test/resources/application-test.yml

```diff
@@ -15,4 +15,10 @@ api:
     url: http://will-be-replaced-by-wireMockServer-bean
 
 dynamodb:
-  print-details-table-name: print-details
\ No newline at end of file
+  print-details-table-name: print-details
+  scheduler-lock-table-name: scheduler-locks
+
+jobs:
```

Comment:
These properties (jobs.print-requests ...) are not required here now if the are not overriding the ones in the main application yml.
But I think I'd like to see the conditional scheduling config and bean that @Neil-Massey has suggested in this PR please?

Reply:
Ok I'll look to add that to this PR then

Reply:
Added